wemake_python_styleguide/violations/consistency.py

```diff
@@ -2102,3 +2104,33 @@ class IterableUnpackingViolation(ASTViolation):
 
     error_template = 'Found an unnecessary iterable unpacking'
     code = 356
+
+
+@final
+class LineCompriseCarriageReturnViolation(TokenizeViolation):
+    """
+    Forbids to use \r (carriage return) in line breaks.
```

Comment:
use double-backquotes to highlight all `\r` and `\n`

Reply:
understand, will fix in a day

Reply:
fixed